YDotNet/Document/Transactions/Transaction.cs

```diff
@@ -121,7 +120,7 @@ public byte[] StateVectorV1()
     public byte[] StateDiffV1(byte[]? stateVector)
     {
         var handle = TransactionChannel.StateDiffV1(
-            Handle, stateVector, (uint)(stateVector?.Length ?? 0), out var length);
+            Handle, stateVector, (uint) (stateVector?.Length ?? 0), out var length);
```

Comment:
I think we should add this spacing setting to editorconfig or something like that, so that we don't override each other settings

Reply:
I just realized the StyleCop support was disabled in my IDE. 🫠

I enabled it and the IDE auto-formatted the file [like this](https://github.com/y-crdt/ydotnet/commit/76cfddc0e621bc0adf83b924028e0450fdf00258). Do you see the same formatting being applied on your side too?

Reply:
Yes, that should work.

Reply:
Great! I'm resolving the discussion, then.